dependencies.py

```diff
@@ -71,18 +73,46 @@
 
 def ensure_everything_installed():
     """Check if DEPENDENCIES are installed and install them if missing."""
+
+    _remove_old_distributions(DEPENDENCIES, _prev_dependencies_target_dir())
+
+    # If required, create deps folder and append to the path
+    target_dir = _dependencies_target_dir(create=True)
+    if str(target_dir) not in sys.path:
+        print(f'Prepending {target_dir} to sys.path')
+        sys.path.insert(0, str(target_dir))
+
+    # pkg_resources needs to be reloaded to be up-to-date with newly installed or removed deps
+    importlib.reload(pkg_resources)
+
+    profile_python_names = [item.name for item in _dependencies_target_dir().iterdir()]
+    print("Contents of our deps dir:\n    %s" % "\n    ".join(profile_python_names))
+
     print("sys.path:")
     for directory in sys.path:
         print("  - %s" % directory)
-    profile_python_names = [item.name for item in _dependencies_target_dir().iterdir()]
-    print("Contents of our profile's python dir:\n    %s" % "\n    ".join(profile_python_names))
+
     _ensure_prerequisite_is_installed()
     missing = _check_presence(DEPENDENCIES)
     if platform.system() == "Windows":
         missing += _check_presence(WINDOWS_PLATFORM_DEPENDENCIES)
         _ensure_h5py_installed()
-    target_dir = _dependencies_target_dir()
-    _install_dependencies(missing, target_dir=target_dir)
+
+    if missing:
```

Comment:
`if` is nogal groot. `if not missing: print()... return` doen? Dan kan de hoofdmoot vier spaties naar links.

Hoewel... als je het volgende review-punt meeneemt is de "if" zal visueel een stuk kleiner, dan is het wel prima.